lib/src/date-input/YearPicker.tsx

```diff
@@ -97,23 +100,20 @@ const YearPickerButton = styled.button<YearPickerButtonPropsType>`
       : ``}
 
   &:hover {
```

Comment:
The font size and line height are being repeated in three different states, causing duplication of code. We can put it all together to avoid this:
```
  &:active, &:hover, &:focus{
    font-size: 1.5rem;
    line-height: 36px;
  }
```